ksqldb-api-client/src/test/java/io/confluent/ksql/api/client/ClientTest.java

```diff
@@ -591,24 +591,6 @@ public void shouldInsertInto() throws Exception {
     assertThat(testEndpoints.getLastTarget(), is("test-stream"));
   }
 
-  @Test
-  public void shouldHandleErrorResponseFromInsertInto() {
```

Comment:
Why are these tests being removed? They are validating that an informative 400 response error message is returned to the user. We can update the exception message to be more realistic (in light of the changes in this PR) but I think we should keep the test coverage.

Reply:
But an "insert into" should be supported now? What kind of error do you have in mind that we should test?

Reply:
The purpose of this test is to check that _any_ error from the insert values endpoint is properly propagated to the user. If you look at the code for the endpoint, you'll see that there are plenty of other errors that could be triggered (besides attempting to insert into a table), e.g., trying to insert into a non-existent source.

Reply:
Deleting this test opens us up to potential regressions where the client does not properly propagate error messages from the server response to the user. We should not delete this coverage.

Reply:
Hi @vcrfxia, do you mean we should also test handling error like these:
https://github.com/confluentinc/ksql/blob/master/ksqldb-rest-app/src/main/java/io/confluent/ksql/api/impl/InsertsStreamEndpoint.java#L61-L80

I agree we should cover those but that seems belong to another PR since these PR is about allowing insert to table. I can create another issue about above missing tests and fix those in a separate PR. What do you think?

Reply:
I'm saying we should have one test to validate that _any_ error is properly propagated from the server to the user (via the client). We do not need to test every possible server error. The proper place for testing each possible server error would be in a unit/integration test for the server endpoint, not the client. The client tests should just contain a single test to validate that any error is properly propagated to the user.

Reply:
I think we should simply update the test highlighted in this comment (in this PR) to test a different error, rather than removing the test in this PR and opening a follow-up PR to add it back.